client/my-sites/stats/hooks/use-notice-visibility-query.ts

```diff
@@ -0,0 +1,21 @@
+import { useQuery } from '@tanstack/react-query';
+import wpcom from 'calypso/lib/wp';
+
+export function queryNotices( siteId: number | null ): Promise< any > {
```

Comment:
```suggestion
export function queryNotices( siteId: number | null ): Promise< Notices > {
```

```
type Notices = {
	new_stats_feedback: boolean;
	opt_in_new_stats: boolean;
	opt_out_new_stats: boolean;
	traffic_page_highlights_module_settings: boolean;
	traffic_page_settings: boolean;
};
```

What do you think about introducing the type `Notices` to demonstrate keys from the API response as well?

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/77867/commits/c13e2ed44ff6d048dd3be9e52b34df74d7bf5c85